model_bakery/random_gen.py

```diff
@@ -326,7 +326,7 @@ def gen_date_range():
     from psycopg2.extras import DateRange
 
     base_date = gen_date()
-    interval = gen_interval()
+    interval = gen_interval(offset=24 * 60 * 60 * 1000)  # force at least 1 day interval
```

Comment:
This is somehow unrelated to this PR but I only discovered this issue after [this CI failure](https://travis-ci.com/github/model-bakers/model_bakery/jobs/363862469#L958).

As you can read in the error message, both `upper` and `lower` dates for the `DateRangeField` were the same. To avoid that, I'm forcing an offset to the interval of 1 day in milliseconds.